examples/zkdetect/main.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
@jdef: Why would you want to execute a separate binary? In any case, you can easily do that by compiling the test binary and then running the example you want: `$ go test -c && ./pkg.test -test.run=ExampleXXX`